iroha/src/torii.rs

```diff
@@ -105,6 +124,23 @@ struct ToriiState {
     consumers: Arc<RwLock<Vec<Consumer>>>,
     system: Arc<RwLock<System>>,
     events_sender: EventsSender,
+    transactions_queue: Arc<RwLock<Queue>>,
+    sumeragi: Arc<RwLock<Sumeragi>>,
+}
+
+impl ToriiState {
+    pub async fn collect_pending_transactions(&self) -> PendingTransactions {
+        self.transactions_queue
+            .read()
+            .await
+            .pending_transactions()
+            .into_iter()
+            .map(|tx| Transaction {
+                payload: tx.payload,
+                signatures: tx.signatures,
+            })
```

Comment:
`tx.into_pending` or just `Into` would help to clean code and make it more type safe.